### DISCUSSION TITLE
URLs using "https" protocol

### DISCUSSION THREAD

**Discussion by agabaldon** [SemanticApplicationDesignLanguage/sadl#609] [Ideas]
There are ontologies out there that use "https" in their URI. Should we allow that in SADL?

**Comment by crapo** [comment]
I am sure the answer is yes. This is done and will be in the next PR.

**Reply by agabaldon** [reply]
Great!

**Comment by crapo** [comment]
@agabaldon , it is worth noting that Jena Rules do not support https protocol so any file with an https URI can't have rules in it without errors ensuing.

**Comment by crapo** [comment]
@agabaldon , let me correct that. A rule cannot reference a concept from any ontology with an https protocol or the rules file will not load. I've posted the issue to the Jena user email list.

**Comment by crapo** [comment]
Jena folks agree. I've created a Jira in Apache Jena. If it becomes an issue I can create a patch.
